added info on how to start influx (#4113)
influxdata/docs-v2: content/influxdb/v2.2/get-started.md

- [Visualize data](#visualize-data)
- [Monitor and alert](#monitor-and-alert)

> **NOTE:** You must start the influxd daemon in order to start working with InfluxDB.  See [InfluxDB service](/influxdb/v2.2/reference/cli/influxd/). For information on the command line interface (CLI) see [InfluxDB command line interface](/influxdb/v2.2/reference/cli/influx/).  Once you've started the influxd daemon, use `localhost:8086` to log in to your instance. 

Reviewer: ```suggestion
*Note:** To run InfluxDB, start the `influxd` daemon ([InfluxDB service](/influxdb/v2.2/reference/cli/influxd/)) using the [InfluxDB command line interface](/influxdb/v2.2/reference/cli/influx/). Once you've started the `influxd` daemon, use `localhost:8086` to log in to your instance.
```

> To start Influx do the following:
>   1. Open a terminal.
>   2. Type influxd in the command line.

Reviewer: ```suggestion
To start InfluxDB, do the following:
  1. Open a terminal.
  2. Type `influxd` in the command line.
```
